yt_dlp/extractor/spankbang.py

```diff
@@ -128,6 +128,12 @@ def extract_format(format_id, format_url):
                     format_url = format_url[0]
                 extract_format(format_id, format_url)
 
+        og_url = self._search_regex(
+            r'<meta property="og:url" content="(.*?)"', webpage, 'og:url', default=None)
```

Comment:
This should probably use one of the `_og_search` functions, like `self._og_search_url`.